packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -26,23 +27,25 @@ export interface ProfilePictureProps
   nativeLazyLoading?: boolean;
 }
 
-const sizeClasses = {
+const sizeClasses: Record<ProfileImageSize, string> = {
   xsmall: 'w-5 h-5',
   small: 'w-6 h-6',
   medium: 'w-8 h-8',
   large: 'w-10 h-10',
   xlarge: 'w-12 h-12',
   xxlarge: 'w-14 h-14',
-  xxxlarge: 'w-24 h-24',
+  xxxlarge: 'w-16 h-16',
```

Comment:
Our guideline states `xxxlarge` to be `w-16 h-16`.

Reply:
Does it have impact on any existing code?
Please tripple check 🙏

Reply:
There was only one implementation which is the profile. Should be covered here 🚢